projects/ng-core-tester/src/app/record/editor/editor.component.ts

```diff
@@ -14,34 +14,38 @@
  * You should have received a copy of the GNU Affero General Public License
  * along with this program.  If not, see <http://www.gnu.org/licenses/>.
  */
-import { Component } from '@angular/core';
-import JSONSchema from './schema.json';
+import { AfterContentChecked, Component, OnChanges, OnInit, SimpleChanges } from '@angular/core';
+import { ActivatedRoute } from '@angular/router';
 
 @Component({
   selector: 'app-editor',
   templateUrl: './editor.component.html'
 })
-export class EditorComponent {
-
-  // editor settings
-  editorSettings = {
-    longMode: true,  // editor long mode
-    template: {
-      recordType: undefined,    // the resource considerate as template
-      loadFromTemplate: false,  // enable load from template button
-      saveAsTemplate: false     // allow to save the record as a template
-    }
-  };
-
-  // JSONSchema
-  schema = {};
-
-  // form initial values
+export class EditorComponent implements OnInit, AfterContentChecked {
+
+  /* form initial values */
   model = {};
 
-  /** Constructor */
-  constructor() {
-    this.schema = JSONSchema;
+  /* Model to display on the top of the form */
+  modelDisplay = {};
+
+  /** Edit or New mode */
+  mode: 'Edit' | 'New' = 'New';
```

Comment:
Need translation?

Reply:
This is only to indicate the mode in the form. No need to translate it.